packages/openactive-integration-tests/test/features/notifications/change-of-logistics-notifications/implemented/change-of-logistics-notifications-test.js

```diff
@@ -0,0 +1,58 @@
+const { FeatureHelper } = require('../../../../helpers/feature-helper');
+const {
+  FlowStageRecipes,
+  FlowStageUtils,
+  TestInterfaceActionFlowStage,
+  OpportunityFeedUpdateFlowStageUtils,
+} = require('../../../../helpers/flow-stages');
+
+FeatureHelper.describeFeature(module, {
+  testCategory: 'notifications',
+  testFeature: 'change-of-logistics-notifications',
+  testFeatureImplemented: true,
+  testIdentifier: 'change-of-logistics-notifications',
+  testName: 'Updating logistics information in after B request.',
+  testDescription: 'ChangeOfLogisticsSimulateAction triggered after B request to update name, startDate, endDate, duration or location properties of Opportunity.',
+  // The primary opportunity criteria to use for the primary OrderItem under test
+  testOpportunityCriteria: 'TestOpportunityBookable',
+  // It is not possible to run multiple OrderItem tests for this due to (https://github.com/openactive/openactive-test-suite/issues/312)
+  skipMultiple: true,
+},
+(configuration, orderItemCriteriaList, featureIsImplemented, logger) => {
+  // ## Initiate Flow Stages
+  const { fetchOpportunities, c1, c2, b, defaultFlowStageParams } = FlowStageRecipes.initialiseSimpleC1C2BFlow(orderItemCriteriaList, logger);
+  const [simulateChangeOfLogisticsUpdate, opportunityFeedUpdate] = OpportunityFeedUpdateFlowStageUtils.wrap({
+    wrappedStageFn: prerequisite => (new TestInterfaceActionFlowStage({
+      ...defaultFlowStageParams,
+      testName: 'TestInterfaceAction (test:ChangeOfLogisticsSimulateAction)',
+      prerequisite,
+      createActionFn: () => ({
+        type: 'test:ChangeOfLogisticsSimulateAction',
+        objectType: fetchOpportunities.getOutput().orderItems[0].orderedItem['@type'],
+        objectId: fetchOpportunities.getOutput().orderItems[0].orderedItem['@id'],
```

Comment:
This and the line above is the change from Luke's changes.

I think I have some fundamental misunderstand of how Test Suite works as this only runs for ScheduledSession and Slots so only simulates those change of logistics (whereas the SessionSeries and FacilityUse can also change). Should I try and simulate the superEvent changing 50% of the time or something? Or a separate test for the superEvent?